Rakefile

```diff
@@ -55,9 +55,11 @@ end
 desc "Test Datadog Process agent"
 task :test do
   cmd = "go list ./... | grep -v vendor | xargs go test"
+  tags = ["secrets"]
```

Comment:
This should only be done for unix (secrets isn't supported for Windows as it runs as root)